### ISSUE TITLE
"Almost done! Fiddling with results…" is not finishing

### ISSUE THREAD

**Issue by HowMuchLonger** [hsoft/dupeguru#537]
I have been running DupeGuru over a large set of folders. Over the process of several days it found over 1.3M duplicates. Now instead of displaying the results window and deleting duplicates, I the progress bar is stuck at the end with a message "Almost done! Fiddling with results…".

DupeGuru has been like this for hours now. There is no process indication. DupeGuru uses <1% CPU. It remains at ca. 1.2 GB RAM as has been the case for most of the duplicate search. There is also no disk activity. So it appears like DupeGuru does nothing. 

I am running this on a Mac with macOS Mojave 10.14.1.

**Comment by HowMuchLonger** [comment]
Some 15 hours later the process has finished. So DupeGuru was indeed doing something and was not failing.

This might still be a case for optimization: There was low consumption of CPU, RAM and disk. More resources were available for DupeGuru to use. So I guess this process could have completed faster. Much faster.

**Comment by HowMuchLonger** [comment]
Some 15 hours later the process has finished. So DupeGuru was indeed doing something and was not failing.

This might still be a case for optimization: There was low consumption of CPU, RAM and disk. More resources were available for DupeGuru to use. So I guess this process could have completed faster. Much faster.

**Comment by DrivenMadz** [comment]
Agree, getting same result. scanning 2 TB drive. Sitting on almost done for 5 hours.

**Comment by l4kr** [comment]
At the very least it would be cool to see SOME output. Even if it was just a wall of text showing which files are processing in the logs.

**Comment by ny-native2017** [comment]
This "almost done.." msg has been up for more than 12 hours. This follows a first pass "scan Folders" where dg got thru 2.5TB of folders but struggling by intermittently "not responding" then coming back. Next went to "scan Content" that has been running for 24 hours, then  "almost done.." msg has been up for more than 12 hours. Wish there was better feedback as to when to give up, start over and trim the folder sizes to work incrementally with smaller "mouth fulls"

**Comment by KIDTREO** [comment]
Agreed - now in early 2025 but with even larger volumes - 8TB with what appears to be billions of files.  Been at "Fiddling" message with no feedback for approx 20 hrs...the good news IG is that CPU is nearly maxed from DupeGurus process so will continue hoping it hasn't hung?

**Comment by billxsand** [comment]
This is still an issue, i realize I lack patience but ANY SMALL indication that something is happening would be helpful here
